src/ophys_etl/modules/segmentation/filter/filter_utils.py

```diff
@@ -0,0 +1,149 @@
+from typing import Optional
+import numpy as np
+
+from ophys_etl.modules.decrosstalk.ophys_plane import OphysROI
+from ophys_etl.modules.segmentation.utils.roi_utils import (
+    select_window_from_background)
+
+
+def mean_metric_from_roi(
```

Comment:
there is a pre-exising very related implementation, and in a different place:
https://github.com/AllenInstitute/ophys_etl_pipelines/blob/8316643127f3e1e3b407546cc64944f63a232367/src/ophys_etl/modules/segmentation/qc_utils/roi_utils.py#L802-L828
can you consolidate into 1 place?

Reply:
I've moved `mean_metric_from_roi` and `median_metric_from_roi` into `utils/roi_utils.py`. The `average_roi_metric` method in `qc_utils/roi_utils` now calls `mean_metric_from_roi`